airbyte-integrations/connectors/source-gitlab/metadata.yaml

```diff
@@ -25,6 +25,15 @@ data:
   releaseStage: generally_available
   releases:
     breakingChanges:
+      3.0.0:
+        message:
+          In this release, merge_request_commits stream was fixed so now it returns commits for each found merge_request.
```

Comment:
In this release, merge_request_commits stream schema has been fixed so that it returns commits for each merge_request. Users will need to refresh the source schema and reset merge_request_commits stream after upgrading.